[CHANGE ME] Re-generated to pick up changes in the API or client library generator.

yoshi-automation opened the pull request:
This PR was generated using Autosynth. :rainbow:

Synth log will be available here:
https://source.cloud.google.com/results/invocations/86ec5a02-193b-4725-a427-0aec843e8c4a/targets

- [ ] To automatically regenerate this PR, check this box.

Source-Link: https://github.com/googleapis/synthtool/commit/da5c6050d13b4950c82666a81d8acd25157664ae
Source-Link: https://github.com/googleapis/synthtool/commit/477764cc4ee6db346d3febef2bb1ea0abf27de52
PiperOrigin-RevId: 336682434
Source-Link: https://github.com/googleapis/googleapis/commit/438e61571a44e1cd58f99cf8a8eb6163d0f0ee39
Source-Link: https://github.com/googleapis/synthtool/commit/9b0da5204ab90bcc36f8cd4e5689eff1a54cc3e4
Source-Link: https://github.com/googleapis/synthtool/commit/f3c04883d6c43261ff13db1f52d03a283be06871
PiperOrigin-RevId: 333125205
Source-Link: https://github.com/googleapis/googleapis/commit/4ede77543c77072a2ac9f78353de2de2f8c4c501
Source-Link: https://github.com/googleapis/googleapis/commit/aaac658367398e478d650768344b88acebad50d9
Source-Link: https://github.com/googleapis/synthtool/commit/27f4406999b1eee29e04b09b2423a8e4646c7e24
Source-Link: https://github.com/googleapis/synthtool/commit/dba48bb9bc6959c232bec9150ac6313b608fe7bd
Source-Link: https://github.com/googleapis/synthtool/commit/257fda18168bedb76985024bd198ed1725485488
Source-Link: https://github.com/googleapis/synthtool/commit/ffcee7952b74f647cbb3ef021d95422f10816fca
Source-Link: https://github.com/googleapis/synthtool/commit/d302f93d7f47e2852e585ac35ab2d15585717ec0
Source-Link: https://github.com/googleapis/synthtool/commit/d91dd8aac77f7a9c5506c238038a26fa4f9e361e
PiperOrigin-RevId: 327026955
Source-Link: https://github.com/googleapis/googleapis/commit/0dc0a6c0f1a9f979bc0690f0caa5fbafa3000c2c
Source-Link: https://github.com/googleapis/synthtool/commit/4f8f5dc24af79694887385015294e4dbb214c352
Source-Link: https://github.com/googleapis/synthtool/commit/bfcdbe0da977b2de6c1c0471bb6dc2f1e13bf669
Source-Link: https://github.com/googleapis/synthtool/commit/39b527a39f5cd56d4882b3874fc08eed4756cebe
Source-Link: https://github.com/googleapis/synthtool/commit/5dfda5621df45b71b6e88544ebbb53b1a8c90214
Source-Link: https://github.com/googleapis/synthtool/commit/ee7506d15daa3873accfff9430eff7e3953f0248
Source-Link: https://github.com/googleapis/synthtool/commit/f07cb4446192952f19be3056957f56d180586055
PiperOrigin-RevId: 320411362
Source-Link: https://github.com/googleapis/googleapis/commit/db69b46790b55a82ab7cfa473d031da787bc7591
PiperOrigin-RevId: 320016414
Source-Link: https://github.com/googleapis/googleapis/commit/aba9760d6abc15f724753c26d6f306e0f76dd11a
Source-Link: https://github.com/googleapis/synthtool/commit/303271797a360f8a439203413f13a160f2f5b3b4
PiperOrigin-RevId: 318028816
Source-Link: https://github.com/googleapis/googleapis/commit/b882b8e6bfcd708042ff00f7adc67ce750817dd0
PiperOrigin-RevId: 317949519
Source-Link: https://github.com/googleapis/googleapis/commit/7157f9552747421572cf1ab3aec1105c05ebd4f9
Source-Link: https://github.com/googleapis/synthtool/commit/cd522c3b4dde821766d95c80ae5aeb43d7a41170
PiperOrigin-RevId: 316182409
Source-Link: https://github.com/googleapis/googleapis/commit/184661793fbe3b89f2b485c303e7466cef9d21a1
PiperOrigin-RevId: 316038172
Source-Link: https://github.com/googleapis/googleapis/commit/e0558da52e48b9f1a4bd5104e6b5e219a48f1dec
PiperOrigin-RevId: 315548189
Source-Link: https://github.com/googleapis/googleapis/commit/ca1372c6d7bcb199638ebfdb40d2b2660bab7b88
Source-Link: https://github.com/googleapis/synthtool/commit/e99975b6b49827b8720f0a885e218dbdb67849ca
Source-Link: https://github.com/googleapis/synthtool/commit/ffe10407ee2f261c799fb0d01bf32a8abc67ed1e
Source-Link: https://github.com/googleapis/synthtool/commit/71b8a272549c06b5768d00fa48d3ae990e871bec
PiperOrigin-RevId: 313460921
Source-Link: https://github.com/googleapis/googleapis/commit/c4e37010d74071851ff24121f522e802231ac86e
PiperOrigin-RevId: 312689208
Source-Link: https://github.com/googleapis/googleapis/commit/dec3204175104cef49bf21d685d5517caaf0058f
Source-Link: https://github.com/googleapis/synthtool/commit/d2364eb80b840a36136c8ce12f1c6efabcc9600e
PiperOrigin-RevId: 312088359
Source-Link: https://github.com/googleapis/googleapis/commit/5a90d467aa65e7f038f87585e8fbb45d74475e7c
Source-Link: https://github.com/googleapis/synthtool/commit/7482e79a82e353248769d819788adc1213e8c207
Source-Link: https://github.com/googleapis/synthtool/commit/09c48461232ce929c34386259eb59018ad2d8eef
PiperOrigin-RevId: 309824146
Source-Link: https://github.com/googleapis/googleapis/commit/e0f9d9e1f9de890db765be46f45ca8490723e3eb

tseaver commented on docs/gapic/v2/api.rst:
```diff
@@ -1,5 +1,5 @@
-Client for Stackdriver Logging API
-==================================
+Client for Cloud Logging API
```
@busunkim96 Is this branding change intended?

simonz130 replied:
Yes, Cloud Logging is a new brand name

tseaver commented on google/cloud/logging_v2/gapic/config_service_v2_client.py:
```diff
@@ -74,58 +74,73 @@ def from_service_account_file(cls, filename, *args, **kwargs):
+        """Return a fully-qualified billing_account string."""
         return google.api_core.path_template.expand(
             "billingAccounts/{billing_account}", billing_account=billing_account,
         )
```
Wow, this is a huge set of potentially breaking changes.

busunkim96 replied:
Yep, lots of breaking changes. I looked at previous iterations of this PR and basically decided it was too much to patch over with synth replaces (path helper removals, path helper renames, method params being reordered).

@simonz130 I'd prefer to continue to let this sit and take the breaking changes when this library moves to the microgenerator. Is that alright with you? I'll also need to sync with you at some point about what to do with the handwritten layer.

tseaver reviewed (changes requested):
This PR is huge, and contains both "housekeeping" changes (adding the docs build) and a bunch of substantive, even breaking changes to the protoc-generated surface.  Please hold of merging it until those issues are hashed out.

yoshi-automation committed:
Integrate Python GAPIC Microgenerator in googleapis. This PR uses using documentai as an example. Depends on https://github.com/googleapis/gapic-generator-python/pull/402

PiperOrigin-RevId: 309824146

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon May 4 15:06:44 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: e0f9d9e1f9de890db765be46f45ca8490723e3eb
Source-Link: https://github.com/googleapis/googleapis/commit/e0f9d9e1f9de890db765be46f45ca8490723e3eb

yoshi-automation committed:
chore(python): create thread_safety doc

This will be included in index.rst files. See https://github.com/googleapis/python-bigtable/pull/26/files

Source-Author: Bu Sun Kim <8822365+busunkim96@users.noreply.github.com>
Source-Date: Tue May 5 10:06:52 2020 -0700
Source-Repo: googleapis/synthtool
Source-Sha: 09c48461232ce929c34386259eb59018ad2d8eef
Source-Link: https://github.com/googleapis/synthtool/commit/09c48461232ce929c34386259eb59018ad2d8eef

yoshi-automation committed:
chore(python): remove extra #!/bin/bash

Co-authored-by: Jeffrey Rennie <rennie@google.com>

Source-Author: Bu Sun Kim <8822365+busunkim96@users.noreply.github.com>
Source-Date: Tue May 12 09:51:12 2020 -0700
Source-Repo: googleapis/synthtool
Source-Sha: 7482e79a82e353248769d819788adc1213e8c207
Source-Link: https://github.com/googleapis/synthtool/commit/7482e79a82e353248769d819788adc1213e8c207

yoshi-automation committed:
fix: restore GAPIC v2 retry configs

Restores GAPIC v2 retry config overrides that were
mistakenly removed during migration.

PiperOrigin-RevId: 312088359

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon May 18 08:57:12 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 5a90d467aa65e7f038f87585e8fbb45d74475e7c
Source-Link: https://github.com/googleapis/googleapis/commit/5a90d467aa65e7f038f87585e8fbb45d74475e7c

yoshi-automation committed:
chore(python): remove one remaining extra bash line

To avoid generating bogus PRs

Co-authored-by: Jeffrey Rennie <rennie@google.com>

Source-Author: Carlos de la Guardia <cguardia@yahoo.com>
Source-Date: Tue May 19 11:57:19 2020 -0500
Source-Repo: googleapis/synthtool
Source-Sha: d2364eb80b840a36136c8ce12f1c6efabcc9600e
Source-Link: https://github.com/googleapis/synthtool/commit/d2364eb80b840a36136c8ce12f1c6efabcc9600e

yoshi-automation committed:
Upgrade protoc-docs-plugin version to 0.7.0 to fix unstable docstrings.

PiperOrigin-RevId: 312689208

Source-Author: Google APIs <noreply@google.com>
Source-Date: Thu May 21 10:00:47 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: dec3204175104cef49bf21d685d5517caaf0058f
Source-Link: https://github.com/googleapis/googleapis/commit/dec3204175104cef49bf21d685d5517caaf0058f

yoshi-automation committed:
bazel: update protobuf, rules_go, gazelle, and gapic-generator-go versions - protobuf v3.12.1 - rules_go v0.23.0 - gazelle v0.21.0 - gapic-generator-go v0.14.1

PiperOrigin-RevId: 313460921

Source-Author: Google APIs <noreply@google.com>
Source-Date: Wed May 27 14:10:16 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: c4e37010d74071851ff24121f522e802231ac86e
Source-Link: https://github.com/googleapis/googleapis/commit/c4e37010d74071851ff24121f522e802231ac86e

yoshi-automation committed:
docs: update python docs template

* docs: update python docs template

* sphinx change

Source-Author: kolea2 <45548808+kolea2@users.noreply.github.com>
Source-Date: Wed May 27 20:44:34 2020 -0400
Source-Repo: googleapis/synthtool
Source-Sha: 71b8a272549c06b5768d00fa48d3ae990e871bec
Source-Link: https://github.com/googleapis/synthtool/commit/71b8a272549c06b5768d00fa48d3ae990e871bec

yoshi-automation committed:
feat: add templates for python samples projects

These templates will be used for templates in python-docs-samples
and in Python client libraries. The README generation code is a modified
version of
https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/scripts/readme-gen.

Co-authored-by: Kurtis Van Gent <31518063+kurtisvg@users.noreply.github.com>

Source-Author: Bu Sun Kim <8822365+busunkim96@users.noreply.github.com>
Source-Date: Thu May 28 14:39:58 2020 -0700
Source-Repo: googleapis/synthtool
Source-Sha: ffe10407ee2f261c799fb0d01bf32a8abc67ed1e
Source-Link: https://github.com/googleapis/synthtool/commit/ffe10407ee2f261c799fb0d01bf32a8abc67ed1e

yoshi-automation committed:
chore: update default ignores for Python libraries

Source-Author: Bu Sun Kim <8822365+busunkim96@users.noreply.github.com>
Source-Date: Thu May 28 14:43:37 2020 -0700
Source-Repo: googleapis/synthtool
Source-Sha: e99975b6b49827b8720f0a885e218dbdb67849ca
Source-Link: https://github.com/googleapis/synthtool/commit/e99975b6b49827b8720f0a885e218dbdb67849ca

yoshi-automation committed:
Regular update of Google API service config with clarification of documentations.

BREAKING CHANGE: `features` field is no longer used. It has been removed from the API runtime for awhile.

PiperOrigin-RevId: 315548189

Source-Author: Google APIs <noreply@google.com>
Source-Date: Tue Jun 9 13:27:00 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: ca1372c6d7bcb199638ebfdb40d2b2660bab7b88
Source-Link: https://github.com/googleapis/googleapis/commit/ca1372c6d7bcb199638ebfdb40d2b2660bab7b88

yoshi-automation committed:
chore: set Ruby namespace in proto options

PiperOrigin-RevId: 316038172

Source-Author: Google APIs <noreply@google.com>
Source-Date: Thu Jun 11 21:09:51 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: e0558da52e48b9f1a4bd5104e6b5e219a48f1dec
Source-Link: https://github.com/googleapis/googleapis/commit/e0558da52e48b9f1a4bd5104e6b5e219a48f1dec

yoshi-automation committed:
fix: use protoc-docs-plugin 0.8.0

Fixes issue with missing newline before 'Attributes' in Python docstrings.

PiperOrigin-RevId: 316182409

Source-Author: Google APIs <noreply@google.com>
Source-Date: Fri Jun 12 14:52:11 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 184661793fbe3b89f2b485c303e7466cef9d21a1
Source-Link: https://github.com/googleapis/googleapis/commit/184661793fbe3b89f2b485c303e7466cef9d21a1

yoshi-automation committed:
fix(python): change autodoc_default_flags to autodoc_default_options

Source-Author: HemangChothani <50404902+HemangChothani@users.noreply.github.com>
Source-Date: Thu Jun 18 22:50:02 2020 +0530
Source-Repo: googleapis/synthtool
Source-Sha: cd522c3b4dde821766d95c80ae5aeb43d7a41170
Source-Link: https://github.com/googleapis/synthtool/commit/cd522c3b4dde821766d95c80ae5aeb43d7a41170

yoshi-automation committed:
chore: update grpc to v1.30.0

PiperOrigin-RevId: 317949519

Source-Author: Google APIs <noreply@google.com>
Source-Date: Tue Jun 23 15:22:22 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 7157f9552747421572cf1ab3aec1105c05ebd4f9
Source-Link: https://github.com/googleapis/googleapis/commit/7157f9552747421572cf1ab3aec1105c05ebd4f9

yoshi-automation committed:
Updates to build Google Ads API build files.

PiperOrigin-RevId: 318028816

Source-Author: Google APIs <noreply@google.com>
Source-Date: Wed Jun 24 02:32:38 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: b882b8e6bfcd708042ff00f7adc67ce750817dd0
Source-Link: https://github.com/googleapis/googleapis/commit/b882b8e6bfcd708042ff00f7adc67ce750817dd0

yoshi-automation committed:
chore: update Py2 support msg to reflect passage of time

Source-Author: Dan O'Meara <danom@google.com>
Source-Date: Mon Jun 29 13:20:01 2020 -0700
Source-Repo: googleapis/synthtool
Source-Sha: 303271797a360f8a439203413f13a160f2f5b3b4
Source-Link: https://github.com/googleapis/synthtool/commit/303271797a360f8a439203413f13a160f2f5b3b4

yoshi-automation committed:
feat: Update Logging API protos.

Renames the service name from Stackdriver Logging to Cloud Logging. Specifies client retry behavior and a few other documentation updates.

PiperOrigin-RevId: 320016414

Source-Author: Google APIs <noreply@google.com>
Source-Date: Tue Jul 7 10:52:01 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: aba9760d6abc15f724753c26d6f306e0f76dd11a
Source-Link: https://github.com/googleapis/googleapis/commit/aba9760d6abc15f724753c26d6f306e0f76dd11a

yoshi-automation committed:
Update protobuf and gapic-generator-csharp versions.

PiperOrigin-RevId: 320411362

Source-Author: Google APIs <noreply@google.com>
Source-Date: Thu Jul 9 09:26:49 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: db69b46790b55a82ab7cfa473d031da787bc7591
Source-Link: https://github.com/googleapis/googleapis/commit/db69b46790b55a82ab7cfa473d031da787bc7591

yoshi-automation committed:
feat(python-library): changes to docs job

* feat(python-library): changes to docs job

* migrate to Trampoline V2
* add docs-presubmit job
* create docfx yaml files and upload them to another bucket

* remove redundant envvars

Source-Author: Takashi Matsuo <tmatsuo@google.com>
Source-Date: Wed Jul 29 16:15:18 2020 -0700
Source-Repo: googleapis/synthtool
Source-Sha: f07cb4446192952f19be3056957f56d180586055
Source-Link: https://github.com/googleapis/synthtool/commit/f07cb4446192952f19be3056957f56d180586055

yoshi-automation committed:
Revert "feat(python-library): changes to docs job (#700)"

This reverts commit f07cb4446192952f19be3056957f56d180586055.

Source-Author: Bu Sun Kim <8822365+busunkim96@users.noreply.github.com>
Source-Date: Wed Jul 29 17:33:57 2020 -0700
Source-Repo: googleapis/synthtool
Source-Sha: ee7506d15daa3873accfff9430eff7e3953f0248
Source-Link: https://github.com/googleapis/synthtool/commit/ee7506d15daa3873accfff9430eff7e3953f0248

yoshi-automation committed:
feat(python-library): changes to docs job

* feat(python-library): changes to docs job

* migrate to Trampoline V2
* add docs-presubmit job
* create docfx yaml files and upload them to another bucket

* remove redundant envvars

* add a failing test first

* fix TemplateSyntaxError: Missing end of comment tag

* serving_path is not needed any more

* use `raw` to make jinja happy

Source-Author: Takashi Matsuo <tmatsuo@google.com>
Source-Date: Thu Jul 30 12:44:02 2020 -0700
Source-Repo: googleapis/synthtool
Source-Sha: 5dfda5621df45b71b6e88544ebbb53b1a8c90214
Source-Link: https://github.com/googleapis/synthtool/commit/5dfda5621df45b71b6e88544ebbb53b1a8c90214

yoshi-automation committed:
fix(python-library): add missing changes

Source-Author: Takashi Matsuo <tmatsuo@google.com>
Source-Date: Thu Jul 30 18:26:35 2020 -0700
Source-Repo: googleapis/synthtool
Source-Sha: 39b527a39f5cd56d4882b3874fc08eed4756cebe
Source-Link: https://github.com/googleapis/synthtool/commit/39b527a39f5cd56d4882b3874fc08eed4756cebe

yoshi-automation committed:
chore(py_library): add split_system_tests

Source-Author: Takashi Matsuo <tmatsuo@google.com>
Source-Date: Fri Jul 31 16:17:13 2020 -0700
Source-Repo: googleapis/synthtool
Source-Sha: bfcdbe0da977b2de6c1c0471bb6dc2f1e13bf669
Source-Link: https://github.com/googleapis/synthtool/commit/bfcdbe0da977b2de6c1c0471bb6dc2f1e13bf669

yoshi-automation committed:
chore(py_library): add some excludes in docs/conf.py

This should fix build failures in python-bigquery.
Example:
https://github.com/googleapis/python-bigquery/pull/205

* also add a comment

Source-Author: Takashi Matsuo <tmatsuo@google.com>
Source-Date: Mon Aug 3 15:08:00 2020 -0700
Source-Repo: googleapis/synthtool
Source-Sha: 4f8f5dc24af79694887385015294e4dbb214c352
Source-Link: https://github.com/googleapis/synthtool/commit/4f8f5dc24af79694887385015294e4dbb214c352

yoshi-automation committed:
Update protobuf workspace dependency to v3.13.0.

PiperOrigin-RevId: 327026955

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Aug 17 08:40:22 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 0dc0a6c0f1a9f979bc0690f0caa5fbafa3000c2c
Source-Link: https://github.com/googleapis/googleapis/commit/0dc0a6c0f1a9f979bc0690f0caa5fbafa3000c2c

yoshi-automation committed:
chore(py-library): enable snippet-bot

Co-authored-by: Benjamin E. Coe <bencoe@google.com>

Source-Author: Takashi Matsuo <tmatsuo@google.com>
Source-Date: Tue Sep 1 17:14:08 2020 +0000
Source-Repo: googleapis/synthtool
Source-Sha: d91dd8aac77f7a9c5506c238038a26fa4f9e361e
Source-Link: https://github.com/googleapis/synthtool/commit/d91dd8aac77f7a9c5506c238038a26fa4f9e361e

yoshi-automation committed:
chore(py-library): update decrypt secrets file

* chore(py-library): update decrypt secrets file

From https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/scripts/decrypt-secrets.sh

* docs: explain conditional

Co-authored-by: Jeffrey Rennie <rennie@google.com>

Source-Author: Bu Sun Kim <8822365+busunkim96@users.noreply.github.com>
Source-Date: Tue Sep 8 11:35:59 2020 -0600
Source-Repo: googleapis/synthtool
Source-Sha: d302f93d7f47e2852e585ac35ab2d15585717ec0
Source-Link: https://github.com/googleapis/synthtool/commit/d302f93d7f47e2852e585ac35ab2d15585717ec0

yoshi-automation committed:
chore(python-library): use sphinx 1.5.5 for the docfx job

Originally tested at:
https://github.com/googleapis/python-texttospeech/pull/89

This change will fix the missing docstring in the yaml files.

Source-Author: Takashi Matsuo <tmatsuo@google.com>
Source-Date: Thu Sep 10 04:12:14 2020 +0000
Source-Repo: googleapis/synthtool
Source-Sha: ffcee7952b74f647cbb3ef021d95422f10816fca
Source-Link: https://github.com/googleapis/synthtool/commit/ffcee7952b74f647cbb3ef021d95422f10816fca

yoshi-automation committed:
build(python): use release-publish app for notifying GitHub of releas…

* build(python): use release-publish app for notifying GitHub of release status

* fix: re-add pypi password

Source-Author: Bu Sun Kim <8822365+busunkim96@users.noreply.github.com>
Source-Date: Wed Sep 16 08:46:42 2020 -0600
Source-Repo: googleapis/synthtool
Source-Sha: 257fda18168bedb76985024bd198ed1725485488
Source-Link: https://github.com/googleapis/synthtool/commit/257fda18168bedb76985024bd198ed1725485488

yoshi-automation committed:
build(python): add secret manager in kokoro

Source-Author: Bu Sun Kim <8822365+busunkim96@users.noreply.github.com>
Source-Date: Wed Sep 16 10:24:40 2020 -0600
Source-Repo: googleapis/synthtool
Source-Sha: dba48bb9bc6959c232bec9150ac6313b608fe7bd
Source-Link: https://github.com/googleapis/synthtool/commit/dba48bb9bc6959c232bec9150ac6313b608fe7bd

yoshi-automation committed:
chore(python): add sphinx doctest extension

Source-Author: Bu Sun Kim <8822365+busunkim96@users.noreply.github.com>
Source-Date: Mon Sep 21 13:09:57 2020 -0600
Source-Repo: googleapis/synthtool
Source-Sha: 27f4406999b1eee29e04b09b2423a8e4646c7e24
Source-Link: https://github.com/googleapis/synthtool/commit/27f4406999b1eee29e04b09b2423a8e4646c7e24

yoshi-automation committed:
feat: Add Java microgen rules to imports

Source-Author: Mira Leung <miraleung@users.noreply.github.com>
Source-Date: Mon Sep 21 18:04:02 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: aaac658367398e478d650768344b88acebad50d9
Source-Link: https://github.com/googleapis/googleapis/commit/aaac658367398e478d650768344b88acebad50d9

yoshi-automation committed:
Publish SLM logging proto for documentation

PiperOrigin-RevId: 333125205

Source-Author: Google APIs <noreply@google.com>
Source-Date: Tue Sep 22 11:39:20 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 4ede77543c77072a2ac9f78353de2de2f8c4c501
Source-Link: https://github.com/googleapis/googleapis/commit/4ede77543c77072a2ac9f78353de2de2f8c4c501

yoshi-automation committed:
chore(python): skip reporting coverage for namespace package

Source-Author: Tres Seaver <tseaver@palladion.com>
Source-Date: Wed Sep 23 10:58:13 2020 -0400
Source-Repo: googleapis/synthtool
Source-Sha: f3c04883d6c43261ff13db1f52d03a283be06871
Source-Link: https://github.com/googleapis/synthtool/commit/f3c04883d6c43261ff13db1f52d03a283be06871

yoshi-automation committed:
chore(python): use BUILD_SPECIFIC_GCLOUD_PROJECT for samples

https://github.com/googleapis/python-talent/blob/ef045e8eb348db36d7a2a611e6f26b11530d273b/samples/snippets/noxfile_config.py#L27-L32

`BUILD_SPECIFIC_GCLOUD_PROJECT` is an alternate project used for sample tests that do poorly with concurrent runs on the same project.

Source-Author: Bu Sun Kim <8822365+busunkim96@users.noreply.github.com>
Source-Date: Wed Sep 30 13:06:03 2020 -0600
Source-Repo: googleapis/synthtool
Source-Sha: 9b0da5204ab90bcc36f8cd4e5689eff1a54cc3e4
Source-Link: https://github.com/googleapis/synthtool/commit/9b0da5204ab90bcc36f8cd4e5689eff1a54cc3e4

yoshi-automation committed:
chore: use python microgenerator rules for logging

PiperOrigin-RevId: 336682434

Source-Author: Google APIs <noreply@google.com>
Source-Date: Mon Oct 12 09:45:22 2020 -0700
Source-Repo: googleapis/googleapis
Source-Sha: 438e61571a44e1cd58f99cf8a8eb6163d0f0ee39
Source-Link: https://github.com/googleapis/googleapis/commit/438e61571a44e1cd58f99cf8a8eb6163d0f0ee39

yoshi-automation committed:
build(python): samples tests should pass if no samples exist

Source-Author: Daniel Sanche <d.sanche14@gmail.com>
Source-Date: Wed Oct 14 08:00:06 2020 -0700
Source-Repo: googleapis/synthtool
Source-Sha: 477764cc4ee6db346d3febef2bb1ea0abf27de52
Source-Link: https://github.com/googleapis/synthtool/commit/477764cc4ee6db346d3febef2bb1ea0abf27de52

yoshi-automation committed:
chore(python_library): change the docs bucket name

Source-Author: Takashi Matsuo <tmatsuo@google.com>
Source-Date: Fri Oct 16 09:58:05 2020 -0700
Source-Repo: googleapis/synthtool
Source-Sha: da5c6050d13b4950c82666a81d8acd25157664ae
Source-Link: https://github.com/googleapis/synthtool/commit/da5c6050d13b4950c82666a81d8acd25157664ae
